doc/source/ray-job-submission/overview.rst

```diff
@@ -0,0 +1,239 @@
+
+
+=========================
+From laptop to production
+=========================
+
+.. warning::
+
+    Ray Job Submission is at Alpha phase with APIs mostly stable but subject to change in the future.
+
+Concepts
+--------
+
+- **Package**: A collection of files and configurations that defines an application, thus allowing it to be executed in a different environment remotely (ideally self-contained).
+
+- **Job**: A Ray application that will be submitted to a Ray cluster for execution. Once a job is submitted, it runs once on the cluster to completion or failure. Retries or different runs with different parameters should be handled by the submitter. Jobs are scoped to the lifetime of a ray cluster.
+
+- **Job Manager**: An entity that manages the lifecycle of a Job and potentially multiple ray clusters, such as scheduling, killing, polling status, getting logs, and persisting inputs / outputs. Should be highly available (HA) by default. Can be any framework with these abilities, such as Airflow.
+
+Within the context of job submission, the packaging part is equivalent to :ref:`Runtime Environments<runtime-environments>`, where we can dynamically configure your desirable ray cluster, actor or task level runtime environment for your submitted job.
+
+**Key inputs for job submission**
+
+- **Entrypoint**: Shell command to execute once your code is unpackaged with runtime environment configured
+    - Typically :code:`python your_script.py`, can also be any shell script such as :code:`echo hello`.
+- **Runtime Environment**:
+    - :code:`working_dir` as local directory: It will be automatically zipped and uploaded to target ray cluster, then unpacked to where your submitted application runs.
+    - :code:`working_dir` as remote URIs, such as s3, git or others, it will be downloaded and unpacked to where your submitted application runs. For details, see :ref:`Runtime Environments<runtime-environments>` for details.
+
+.. note::
+  **The goal of Ray job submssion is to provide a lightweight mechanism for user to submit their locally developed and tested application to a running local / remote ray cluster, thus enabling the user to package, deploy, and manage their ray application as jobs. These jobs can be submitted by a job manager of their choice.**
```

Comment:
I wonder if this belongs in a note or should just go at the very top of the page without being in a note.  It probably doesn't need to be in all bold

Reply:
```suggestion
  **The goal of Ray job submission is to provide a lightweight mechanism for user to submit their locally developed and tested application to a running local / remote ray cluster, thus enabling the user to package, deploy, and manage their ray application as jobs. These jobs can be submitted by a job manager of their choice.**
```

Reply:
Probably would remove bold from this note

Reply:
addressed